stackimage/priming-app/build.gradle

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021 IBM Corporation and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+group = 'dev.odo.java-openliberty.samples'
+description = 'OpenLiberty Application Stack Primer'
+
+// Repositories to use.
+repositories {
+    mavenCentral()
+}
+
+// Plugins.
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+// Java properties.
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+compileJava.options.encoding = 'UTF-8'
+
+// Software dependencies.
+dependencies {
+    providedCompile 'org.eclipse.microprofile:microprofile:3.3'   
+    providedCompile 'jakarta.platform:jakarta.jakartaee-api:8.0.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.6.2'
+    testImplementation 'org.microshed:microshed-testing-liberty:0.9'
```

Comment:
I guess we could remove this..

Reply:
Note that removing MST is a separate piece of work that is not part of the base stack. We can do that work here and port it to the base stack, but i would rather this be done in a separate issue. 
I will open two issues for this and another for enhancing test/stack-test-inner-loop.sh by adding "odo test". The stack-test-inner-loop.sh update has a pre-req adding gradle support to the application-stack-intro